From 63170cfcfbf483716ce2599e165db80713fc2706 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 12 Apr 2022 06:47:19 -0700
Subject: [PATCH] include utility header

Needed for exchange from std namespace

Fixes
dbus-cxx/variant.cpp:135:25: error: 'exchange' is not a member of 'std'

Upstream-Status: Submitted [https://github.com/dbus-cxx/dbus-cxx/pull/98]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 dbus-cxx/variant.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dbus-cxx/variant.cpp b/dbus-cxx/variant.cpp
index bcf2684..97b622c 100644
--- a/dbus-cxx/variant.cpp
+++ b/dbus-cxx/variant.cpp
@@ -11,6 +11,7 @@
 #include <dbus-cxx/dbus-cxx-private.h>
 #include <dbus-cxx/signatureiterator.h>
 #include <stdint.h>
+#include <utility>
 #include "enums.h"
 #include "path.h"
 #include "signature.h"
-- 
2.35.1

